pom.xml

```diff
@@ -342,6 +342,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-jar-plugin</artifactId>
+                <version>2.4</version>
```

Comment:
This is to get rid of the warning that maven complains about when building zalenium

Reply:
👍